src/diagnostics/logPointsSessionWizard.ts

```diff
@@ -345,7 +346,11 @@ class StartDebugAdapterStep extends WizardStep {
             instanceId: this._wizard.selectedInstance.name,
             sessionId: this._wizard.sessionId,
             debugId: this._wizard.debuggerId
-        });
+        };

```

Comment:
This seems to be temporary debug code, `instanceId`, `sessionId` and `debugId` are not useful to end user and have never been in the picture.

Reply:
I thought this could be useful to have for troubleshooting/diagnostic issues. If you don't think this is useful for the users I can put these flags under some #if _DEBUG_

Reply:
We don't really need this either way. The same information will be shown in the DEBUG CONSOLE when debugger starts. It's the payload of the first debug protocol message. Suggest remove altogether.

Reply:
You are right, removing this change...